src/main/java/com/datasonnet/XMLFormatPlugin.java

```diff
@@ -65,6 +67,23 @@ public Value read(String inputXML, Map<String, Object> params) throws Exception
 
     public String write(Value inputXML, Map<String, Object> params) throws Exception {
         JSONObject input = new JSONObject(UjsonUtil.jsonObjectValueTo(inputXML));
+
+        if (params.containsKey(ROOT_ELEMENT)) {
+            JSONObject root = new JSONObject();
+            root.put(params.get(ROOT_ELEMENT).toString(), input);
```

Comment:
throwing a `.toString` on this may result in some very strange output but it's fine for now I think